### DISCUSSION TITLE
Asking for best practice using Admin page just local but sure Login page public

### DISCUSSION THREAD

**Discussion by ClaudioBsh** [fief-dev/fief#350] [Q&A]
I like to ask for security reasons what is best practice how to configure FIEF-Env-Variables to make sure that a URL call to the admin page will be possible just from local network only but sure the loginpage will be accessible public?

**Comment by frankie567** [comment]
Currently, there is no way to block the admin page. Note that only users with the **Fief Administrator** permission have access to the admin dashboard; so it's safe to expose it.

If you really want to block it, you probably can configure specific rules using a reverse proxy.
